### ISSUE TITLE
Old module use ImportModule

### ISSUE THREAD

**Issue by okom3pom** [PrestaShop/autoupgrade#80]
**Describe the bug**
Old module try to access to ImportModule classe

**Upgrade environment (please complete the following information):**
 - Version you started from: 1.6.1.10
 - Version you upgrade to: 1.7.3.3
 - Way of upgrade: Web
 - Step concerned: module

**Displayed log**
```
AH01071: Got error 'PHP message: PHP Fatal error: Class 'ImportModule' not found in /var/www/vhosts/chafoins.com/httpdocs/presta2/modules/importerosc/importerosc.php on line 30\n', referer: https://www.chafoins.com/presta2/adminxxxx/index.php?controller=AdminSelfUpgrade&conf=6&token=7a843423af6472810924dfaf3bba77fe
```

**Expected behavior**
I have an old module in my folder not installed

**Server (please complete the following information):**
 - Web Server: Apache
 - PHP Version: 7.0.30
 - Module Version 4.0.0 beta

**Additional context**

Informations sur votre serveur Linux #1 SMP Tue Jun 30 19:37:11 CEST 2015 x86_64
Version du logiciel serveur Apache
Version de PHP 7.0.30
Limite de mémoire 1024M
Temps d'exécution maximal (max_execution_time) 0
Version de PrestaShop 1.6.1.10
URL de votre boutique http://www.chafoins.com/presta2/
Nom du thème utilisé esprit

**Comment by Quetzacoalt91** [comment]
Hi @okom3pom,

Loading modules is a side effect we want to get rid of. On our side, it happened when we want to disable them for instance.

Do you remember precisely when it happened? It could be during:
* the DB upgrade, when we try to disable non-native modules ; To be fixed with PrestaShop/autoupgrade#86 
* the modules upgrade, when we try to init the catalog.

**Comment by ghost** [comment]
I try test the PR today with more investigation.

**Comment by ghost** [comment]
The PR don't fixe the issue.

It's during the modules upgrades. 

`[DEV] ajax request : upgradeModules
[INTERNAL] /var/www/vhosts/chafoins.com/httpdocs/presta2/modules/blockrss/blockrss.php line 30 - include_once(/var/www/vhosts/chafoins.com/httpdocs/presta2/classes/../tools/pear/PEAR.php): failed to open stream: No such file or directory
[INTERNAL] /var/www/vhosts/chafoins.com/httpdocs/presta2/modules/blockrss/blockrss.php line 30 - include_once(): Failed opening '/var/www/vhosts/chafoins.com/httpdocs/presta2/classes/../tools/pear/PEAR.php' for inclusion (include_path='/var/www/vhosts/chafoins.com/httpdocs/presta2/tools/htmlpurifier/standalone:/var/www/vhosts/chafoins.com/httpdocs/presta2/vendor/pear/pear_exception:/var/www/vhosts/chafoins.com/httpdocs/presta2/vendor/pear/console_getopt:/var/www/vhosts/chafoins.com/httpdocs/presta2/vendor/pear/pear-core-minimal/src:/var/www/vhosts/chafoins.com/httpdocs/presta2/vendor/pear/archive_tar:.:/opt/plesk/php/7.0/share/pear')
[INTERNAL] /var/www/vhosts/chafoins.com/httpdocs/presta2/modules/blockrss/blockrss.php line 31 - include_once(/var/www/vhosts/chafoins.com/httpdocs/presta2/tools/pear_xml_parser/Parser.php): failed to open stream: No such file or directory
[INTERNAL] /var/www/vhosts/chafoins.com/httpdocs/presta2/modules/blockrss/blockrss.php line 31 - include_once(): Failed opening '/var/www/vhosts/chafoins.com/httpdocs/presta2/tools/pear_xml_parser/Parser.php' for inclusion (include_path='/var/www/vhosts/chafoins.com/httpdocs/presta2/tools/htmlpurifier/standalone:/var/www/vhosts/chafoins.com/httpdocs/presta2/vendor/pear/pear_exception:/var/www/vhosts/chafoins.com/httpdocs/presta2/vendor/pear/console_getopt:/var/www/vhosts/chafoins.com/httpdocs/presta2/vendor/pear/pear-core-minimal/src:/var/www/vhosts/chafoins.com/httpdocs/presta2/vendor/pear/archive_tar:.:/opt/plesk/php/7.0/share/pear')`

And the error : 

`[INTERNAL] /var/www/vhosts/chafoins.com/httpdocs/presta2/modules/shopimporter/shopimporter.php line 30 - Class 'ImportModule' not found`

All shops coming from 1.5, have these modules if they don't clean the folder << modules >>.

Remove the module before the upgrade is a way i found two pestashop modules using this class : shopimporter and importerosc

You can find the modules in an archive 1.5 : https://download.prestashop.com/download/releases/prestashop_1.5.6.1.zip

**The modules are not installed.**

**Comment by Quetzacoalt91** [comment]
Yeah I can imagine what the issue is. Using the ModuleRepository will require all module available on the `modules/` folder to get its properties. If one of them has something incompatible with the current core, it fail throw a fatal error although it is uninstalled.
